pylint_django/checkers/__init__.py

```diff
@@ -11,3 +12,4 @@ def register_checkers(linter):
     linter.register_checker(DjangoInstalledChecker(linter))
     linter.register_checker(JsonResponseChecker(linter))
     linter.register_checker(FormChecker(linter))
+    linter.register_checker(MissingBackwardsMigrationChecker(linter))
```

Comment:
@carlio this whole PR is for something I've identified in a project of mine. However I'm wondering whether or not we want this entire checker to be enabled by default or have people enabled it manually as an extension. What do you think? 

NB: depending on the answer this PR may need some adjustments like moving duplicate functions around, renaming modules, etc.

Reply:
Note2: IMO this should go together with `db_performance.py` (and we rename the module) because the test suite is structured around 2 items: tests for regular checkers and tests for checkers around migrations which are explicitly enabled. I woudn't want to duplicate that if possible.

Reply:
@brymut I have made up my mind. Move the new checker class inside the `db_performance` module. That will make it easier with the automation tests as well. After we merge this PR I will take care of renaming things and updating the docs.

Reply:
@atodorov I agree I think it makes sense to group it together with the `db_performance` module as it seems related but I'm easy either way.